pactas_itero.gemspec

```diff
@@ -1,37 +1,37 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'pactas_itero/version'
+require "pactas_itero/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'pactas_itero'
+  spec.name          = "pactas_itero"
   spec.version       = PactasItero::VERSION
-  spec.authors       = ['Simon Fröhler']
+  spec.authors       = ["Simon Fröhler"]
   spec.email         = "simon@shipcloud.io"
-  spec.summary       = %q{pactas_itero provides a client mapping for accessing
-    the Pactas Itero API.}
-  spec.description   = %q{pactas_itero provides a client mapping for accessing
-    the Pactas Itero API, making it easy to post your data to, adn read your
-    data from your Pactas account.}
-  spec.homepage      = 'https://github.com/webionate/pactas_itero'
-  spec.license       = 'MIT'
+  spec.summary       = 'pactas_itero provides a client mapping for accessing
+    the Pactas Itero API.'
+  spec.description   = 'pactas_itero provides a client mapping for accessing
+    the Pactas Itero API, making it easy to post your data to, and read your
+    data from your Pactas account.'
+  spec.homepage      = "https://github.com/webionate/pactas_itero"
```

Comment:
maybe we can create a follow-up story to update these outdated description?

Reply:
Yes, good idea. Will do! :+1: